src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import duke.entity.*;
```

Comment:
Even though you may want to import all the packages, it is still more appropriate to list all the classes you imported from your package, instead of using `*`.